Readme.md

```diff
@@ -22,6 +22,10 @@ To start a local Ethereum node with the contracts deployed, execute:
 
     npm start
 
+You can run Certora's specs with:
+
+    npm run verify
+
```

Comment:
Sooo, is this really true? From looking at the CI setup, I believe there are some dependencies required to run this. For example, some dependencies like `pip3 install certora-cli==7.6.3` are not installed with `npm install`. Could you please add to the Readme the list of necessary installations to run this?

Reply:
Correct. There's a requirement to have certora installed locally.
Will update the docs.

Reply:
Done here https://github.com/codex-storage/codex-contracts-eth/pull/113/commits/f0c7a17e8911915f898df7d424c56b7bc7537220

Reply:
Can you include that commit to this PR? Also, I believe you need Java installed otherwise the Certora won't work, right?